src/main/java/org/folio/consortia/utils/HelperUtils.java

```diff
@@ -17,4 +25,9 @@ public static String randomString(Integer noOfString) {
     RandomStringGenerator generator = new RandomStringGenerator.Builder().withinRange('a', 'z').build();
     return generator.generate(noOfString);
   }
+
+  public static ObjectNode setSourceAsConsortium(JsonNode payload) {
+    String source = FOLIO_SOURCE_VALUE.equalsIgnoreCase(payload.get("source").asText()) ? CONSORTIUM_FOLIO: CONSORTIUM_MARK;
```

Comment:
I think we should be more explicit here to map folio -> CONSORTIUM-FOLIO and marc -> CONSORTIUM-MARC. if other source values are introduced in the future, it would default to CONSORTIUM-MARC and that wont be ideal. Probably should return an error if we find a source value that is not recognized.

Reply:
1. Do you mean this structure

- if source is folio, then it will be CONSORTIUM-FOLIO
- If source is marc, then it will be CONSORTIUM-MARC.
- all other type will throw exceptino
![image](https://github.com/folio-org/mod-consortia/assets/113523904/a87b328f-4c22-461a-a9e3-27edc3851768)

2. What about instance? is it still valid scenario for sharing instance (this code is used by share instance method also)

Reply:
1. The screenshot looks incorrect to me because it is saying if source = CONSORTIUM_FOLIO, set source to CONSORTIUM_FOLIO. But the words are what I expect.

2. I was not aware that this was the case. it was my impression that we would save "consortium" instead of "consortium-folio"  for settings. I was thinking about my initial comment in the instance context instead of settings. 

Generally, what I am looking for is that CONSORTIUM-MARC is not the default output when source does not match CONSORTIUM-FOLIO.

Reply:
Yes agree, @azizbekxm for instances if source = "folio" - result should be "CONSORTIUM_FOLIO", if source = "marc" - result should be "CONSORTIUM_MARC"
For Settings it always should be "consortium"

Reply:
@okolawole-ebsco maybe we can use lower case as "consortium_folio" or "consortium_marc" to make it consistent?

Reply:
@azizbekxm handling such situation using flag isSetting is not approprtiate here, you can leave constants in utility class and move business logic to appropriate service

Reply:
Fixed big thanks